ibis/expr/format.py

```diff
@@ -203,10 +203,10 @@ def _format_column(self, expr):
             self.memo.observe(parent, formatter=self._format_node)
 
         table_formatted = self.memo.get_alias(parent)
-        table_formatted = self._indent(table_formatted)
+        table_formatted = table_formatted
 
         type_display = self._get_type_display(self.expr)
-        return "Column[{}] '{}' from table\n{}".format(
+        return "Column[{}] '{}' from {}".format(
```

Comment:
```suggestion
        return "Column[{}] {!r} from {}".format(
```

Reply:
Will the `repr` of the column name ever be different from its `str`?

Reply:
I suppose that's possible, but very unlikely.